Dockerfile

```diff
@@ -0,0 +1,61 @@
+FROM cloudposse/terraform-root-modules:0.3.3 as terraform-root-modules
```

Comment:
yea, waiting on this PR to be approved https://github.com/cloudposse/terraform-root-modules/pull/13
which will become `0.3.3`

Reply:
@tamsky 
this PR was approved, merged and tagged `0.3.3`:
https://github.com/cloudposse/terraform-root-modules/pull/13

This `init` branch is WIP.
When it's merged into `master`, it will have all the correct references (as it does now, but it's still WIP and might change).

thank you for the review